README.md

```diff
@@ -1,6 +1,14 @@
 # AI Multimodal Public Demos
 
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientist from [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/) and [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/)
+Current contributors working in this project are: Dylan McReynolds and Jospeh Kleinhenz from LBL, and Dan Allan and Juan Marulanda from BNL.
+Our goal is to create database for XAS data that can be used by scientist from beamlines at different Synchrtron Light Sources around the world. They will be able to use this database as a platform to share and use data to improve their research. 
```

Comment:
I actually don't know if this is a goal.

Reply:
This is definitely something that other related projects are doing, and it seems like something that we _could_ do, but it starts to sound like straying in Materials Project territory. I think the focus of this particular effort is to provide centralize access to _multimodal_ data sets from a variety of facilities in order to support multi-modal AI/ML analysis.